Refactoring DataBrain branding (#2234)
segmentio/segment-docs: src/connections/destinations/catalog/databrain/index.md

redirect_from: '/connections/destinations/catalog/supervised-ai/'
---

> [DataBrain](https://usedatabrain.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is a modern GTM tool for product-led companies. DataBrain helps Sales and Customer Success teams close more deals, adopt more product and retain more customers. It ingests data from Segment and leverages machine learning techniques to send users automated alerts, and predict churn, conversion, retention on any event you need to track.

Reviewer: ```suggestion
[DataBrain](https://usedatabrain.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is a tool for product-led companies that helps Sales and Customer Success teams close deals, adopt products, and retain customers through data and analytics. It ingests data from Segment and leverages machine learning techniques to send users automated alerts, and predict churn, conversion, and retention on any event you need to track.
```
